TRACE_SYSTEM_STRING is not need in each trace file anymore.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Upstream Status: Pending
--- driver/src/devicedrv/mali/linux/mali_linux_trace.h
+++ b/linux/mali_linux_trace.h
@@ -13,12 +13,10 @@
 
 #include <linux/types.h>
 
-#include <linux/stringify.h>
 #include <linux/tracepoint.h>
 
 #undef  TRACE_SYSTEM
 #define TRACE_SYSTEM mali
-#define TRACE_SYSTEM_STRING __stringfy(TRACE_SYSTEM)
 
 #define TRACE_INCLUDE_PATH .
 #define TRACE_INCLUDE_FILE mali_linux_trace
